infrastructure/bin/setup_supervisor.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# This script is run during the deployment to set up the supervisor service
+
+cp /home/site/wwwroot/infrastructure/conf/supervisord.conf /etc/supervisor/
+cp /home/site/wwwroot/infrastructure/conf/laravel-worker.conf /etc/supervisor/conf.d/
+supervisord -c /etc/supervisor/supervisord.conf
```

Comment:
The workers are set as autostart so this launches them.  Run `supervisorctl status` to check on them.